libraries/I2S/src/I2S.h

```diff
@@ -30,11 +34,14 @@ class I2S : public Stream {
 
     bool setBCLK(pin_size_t pin);
     bool setDATA(pin_size_t pin);
+    bool setMCLK(pin_size_t pin); // RP - also enables MCLK output
```

Comment:
No need to comment with your initials, the `git blame` will let everyone know any bugs here are your fault, not mine.  :laughing: